src/app/images.tsx

```diff
@@ -9,6 +9,7 @@ export default function Images({
     setDevicePaths
 }: any) {
     const DEBUG = true;
+    const valid_extensions = ["jpg", "jpeg", "tif", "tiff", "JPG", "JPEG", "TIF", "TIFF", "CR2"]
```

Comment:
This is the list of extensions I think we're supporting now (as of this morning, since we just decided to include raw formats from wikipedia): 
```
[
        "jpg", "jpeg", "3fr", "ari", "arw", "bay", "braw", "crw", "cr2", "cr3", "cap", "data",
        "dcs", "dcr", "dng", "drf", "eip", "erf", "fff", "gpr", "iiq", "k25", "kdc", "mdc", "mef",
        "mos", "mrw", "nef", "nrw", "obm", "orf", "pef", "ptx", "pxn", "r3d", "raf", "raw", "rwl",
        "rw2", "rwz", "sr2", "srf", "srw", "tif", "tiff", "x3f",
]
```

I can add them in later if you don't want to include them now, but didn't want to create merge conflicts since I knew you were working on the frontend image selection.

As an idea: it might be easier to convert everything to lowercase when checking the extensions, that way you don't have to worry about case sensitivity. That's what I started doing on the backend anyway.